src/main/java/net/sf/mzmine/modules/peaklistmethods/normalization/standardcompound/StandardCompoundNormalizerTask.java

```diff
@@ -190,10 +195,7 @@ public void run() {
 
 			Feature standardPeak = standardRow.getPeak(file);
 			if (standardPeak == null) {
-			    // What to do if standard peak is not
-			    // available? (Currently this is ruled out by the
-			    // setup dialog, which shows only peaks that are
-			    // present in all samples)
+			    // What to do if standard peak is not available?
```

Comment:
The setup dialog does not only show peaks which are present in all samples, which is why the null check is needed. But should the dialog only show the peaks which are present in all samples?